conv/doc.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 go-swagger maintainers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package conv exposes utilities to convert types.
+//
+// Convert and format functions are essentially a shorthand to `strconv` functions,
```

Comment:
```suggestion
// [Convert] and format functions are essentially a shorthand to `strconv` functions,
```

godoc link is interesting here

Reply:
sure

Reply:
but these are "families" of functions... Not sure the link would point anywhere. Anyway, I've to reformulate.